apps/site/scripts/orama-search/sync-orama-cloud.mjs

```diff
@@ -1,18 +1,16 @@
-import { CloudManager } from '@oramacloud/client';
+import { OramaCloud } from '@orama/core';
 
 import { siteContent } from './get-documents.mjs';
 import { ORAMA_SYNC_BATCH_SIZE } from '../../next.constants.mjs';
 
 // The following follows the instructions at https://docs.orama.com/cloud/data-sources/custom-integrations/webhooks
 
-const INDEX_ID = process.env.ORAMA_INDEX_ID;
-const API_KEY = process.env.ORAMA_SECRET_KEY;
-
-const oramaCloudManager = new CloudManager({
-  api_key: API_KEY,
+const orama = new OramaCloud({
+  projectId: process.env.NEW_ORAMA_PROJECT_ID || '',
+  apiKey: process.env.NEW_ORAMA_API_KEY || '',
```

Comment:
I suspect the GitHub Actions workflow needs to be updated to expose these new env vars?